src/components/application_manager/include/application_manager/commands/hmi/sdl_activate_app_request.h

```diff
@@ -40,13 +40,13 @@ namespace application_manager {
 
 namespace commands {
 
-typedef std::pair<ApplicationSharedPtr, std::vector<ApplicationSharedPtr> >
-    DevicesApps;
 /**
  * @brief SDLActivateAppRequest command class
  **/
 class SDLActivateAppRequest : public RequestFromHMI {
  public:
+  typedef std::vector<application_manager::ApplicationSharedPtr> SDL4Apps;
```

Comment:
@dev-gh Good idea to use typedef. But for the people who is not aware of SDL  it will be ambiguous. May be add undersore or something like this: `SDL_v4_Apps`. Because `4` can be treated as `for` or just as number of apps but not SDL version.